apps/prairielearn/src/lib/features/index.ts

```diff
@@ -13,6 +13,8 @@ const featureNames = [
   'ai-question-generation-course-toggle',
   // Can be applied to any context.
   'ai-question-generation',
+  // Implies lti 1.1. Can be applied to any context.
+  'lti',
```

Comment:
`lti` is super vague given that LTI 1.3 is very much a thing and is not feature-flagged. Why not name this `lti1.1`?

Reply:
In paths or filenames in PL, LTI 1.1 has just been "lti", which an emphasis on making anything new "lti13". I kept the feature flag with that convention, but don't mind changing it to something more explicit.

Reply:
Do you prefer `lti11` or `lti1.1` for the feature flag?